BitwardenShared/UI/Auth/CompleteRegistration/PreventAccountLock/PreventAccountLockView.swift

```diff
@@ -13,98 +13,94 @@ struct PreventAccountLockView: View {
     // MARK: View
 
     var body: some View {
-        VStack {
-            VStack(spacing: 0) {
-                instructionsView
-
-                hintInstructionsView
-
-                writeDownInstructions
+        VStack(spacing: 24) {
+            instructionsView
+
+            ContentBlock(dividerLeadingPadding: 48) {
+                rowView(
+                    image: Asset.Images.lightbulb24,
+                    title: Localizations.createAHint,
+                    subtitle: Localizations.yourHintWillBeSentToYouViaEmailWhenYouRequestIt
+                )
+
+                rowView(
+                    image: Asset.Images.pencil24,
+                    title: Localizations.writeYourPasswordDown,
+                    subtitle: Localizations.beCarefulToKeepYourWrittenPasswordSomewhereSecretAndSafe
+                )
             }
-            .background(Asset.Colors.backgroundSecondary.swiftUIColor)
-            .clipShape(RoundedRectangle(cornerRadius: 10))
         }
         .scrollView()
         .navigationBar(title: Localizations.preventAccountLockout, titleDisplayMode: .inline)
         .toolbar {
-            cancelToolbarItem {
+            closeToolbarItem {
                 store.send(.dismiss)
             }
         }
     }
 
-    // MARK: Private views
+    // MARK: Private
 
     /// The main instructions.
     private var instructionsView: some View {
-        VStack(spacing: 0) {
-            VStack(alignment: .leading, spacing: 8) {
-                Text(Localizations.neverLoseAccessToYourVault)
-                    .styleGuide(.title3, weight: .semibold)
-
-                Text(Localizations.theBestWayToMakeSureYouCanAlwaysAccessYourAccountIsToSetUpSafeguardsFromTheStart)
-                    .styleGuide(.body)
-            }
-            .foregroundStyle(Asset.Colors.textPrimary.swiftUIColor)
-            .padding(24)
+        VStack(spacing: 8) {
+            Text(Localizations.neverLoseAccessToYourVault)
+                .styleGuide(.title2, weight: .semibold)
 
-            Divider()
+            Text(Localizations.theBestWayToMakeSureYouCanAlwaysAccessYourAccountIsToSetUpSafeguardsFromTheStart)
+                .styleGuide(.body)
+                .multilineTextAlignment(.center)
```

Comment:
🎨 I think this `multilineTextAlignment` modifier should apply to both the title and subtitle.

<img width="418" alt="Screenshot 2024-10-21 at 1 16 44 PM" src="https://github.com/user-attachments/assets/19145efc-37a2-4376-9a15-b4cbc6cff17c">

Reply:
good catch!